### DISCUSSION TITLE
Official communication channels

### DISCUSSION THREAD

**Discussion by ZhaoFancy** [01-ai/Yi#51] [Polls]
Let us know which channel you prefer to communicate with our developers!

**Comment by njflove** [comment]
WeChat

**Comment by nomagick** [comment]
WeChat sucks shit.
It doesn't save any chat history and limits the maximum group size.

**Comment by playmsg** [comment]
telegram!

**Comment by sci-m-wang** [comment]
Discord, TG, or QQ, plz! WeChat is really annoying!

**Comment by bilal-aamer** [comment]
Either Telegram or Discord would be better since they are widely used.

**Comment by richardlinq** [comment]
Dear Yi Community Members,

Based on the current voting results (with WeChat accounting for 62%), we have established a WeChat Group specifically for our Chinese-speaking users, to facilitate real-time discussions among Yi's Chinese community.

For instructions on how to join the WeChat Group, please visit (in Chinese): https://github.com/01-ai/Yi/issues/43#issuecomment-1827285245

For **non-Chinese** speakers and those who **do not use WeChat**, please continue to **VOTE** and express your preferences. This will help us better understand everyone's preferred communication channels. Thank you!

<img width="666" alt="image" src="https://github.com/01-ai/Yi/assets/1932744/580ded7b-9916-4fe1-bf54-1f93791d5fc6">

**Reply by bilal-aamer** [reply]
Hey @richardllin, will do however ig it would be a much better estimation to poll with multiple choice than the current single choice for voting.

**Reply by richardlinq** [reply]
Hey @bilal-aamer 
I don't know how to make it multiple-choice. It seems like this feature isn't available? 
Or do you have any recommended multiple-choice tools for developers?
